docs/v1/P4Runtime-Spec.mdk

```diff
@@ -2574,24 +2575,27 @@ An `ActionProfileMember` entity update message has the following fields:
 * `action` is the specification of the P4 action instance bound to the action
   profile member entry.
 
-An action profle member may be inserted, modified or deleted as per the
+An action profile member may be inserted, modified or deleted as per the
 following semantics.
 
 * `INSERT`: Add a new member entry bound to an eligible P4 action
   specification. The member id must be different from ids of already programmed
-  entries for that extern. The action specification must be provided. The total
-  number of members should not exceed the maximum specified in the P4 extern
-  specification, as a result of this insertion.
+  entries for that extern, or the server must return an `ALREADY_EXISTS` error
+  code. The action specification must be provided or the server must return
+  `INVALID_ARGUMENT`. The total number of members should not exceed the maximum
+  specified in the P4 extern specification as a result of this insertion or the
+  server should return `RESOURCE_EXHAUSTED`.
 * `MODIFY`: Modify the action specification of an existing member entry. An
-  entry with the member id must exist and the action specification must be
-  provided. If the member is part of an action profile group, then the bound
-  action must be of the same action type as actions bound to other members of
-  the group. See the condition on bound actions for members belonging to a
-  group.
-* `DELETE`: Delete the member entry and deallocate the member id. The member
-  should not be part of an action profile group. If needed, the action profile
-  group should first be modified to remove the member from the group. The member
-  should not be referenced in the table action of any table entry.
+  entry with the member id must exist (or the server must return `NOT_FOUND`)
+  and the action specification must be provided (or the server must return
```

Comment:
Style nit - for MODIFY, the server's error responses are in parens, whereas they flow with the text in INSERT & DELETE.

Reply:
Will fix (remove the parens)